Test failure in test_share_images_cache on fast computers

When running the tests I have a failure in test.image.test_load_image.test_share_images_cache:

```
>       assert duration.seconds > 0.25                               
E       assert 0.19840230899899325 > 0.25
```

I've noticed that that value has been recently changed, from 0.3 to 0.25 in commit 463a9e3a with the note “Test systems are faster than they used to”.

If I understand correctly the intent of the test, the important thing here is that the second execution takes less time than the first, so I've run the test locally with the following patch

```
 test/image/test_load_image.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/image/test_load_image.py b/test/image/test_load_image.py
index ef19221..508f65e 100644
--- a/test/image/test_load_image.py
+++ b/test/image/test_load_image.py
@@ -63,8 +63,8 @@ def test_share_images_cache(tmp_path):
 
     with time_execution() as duration:
         build_pdf_with_big_images()
-    assert duration.seconds > 0.25
+    first_time_duration = duration.seconds
 
     with time_execution() as duration:
         build_pdf_with_big_images()
-    assert duration.seconds < 0.3
+    assert duration.seconds < first_time_duration / 2
```

could it be a valid alternative (maybe with a different factor) that doesn't require updating every time the test systems get faster?

Please let me know if I have to turn this into a PR (I'm not really used to the github interface, but if I have to I can re-learn how to use it)

--- a/test/image/test_load_image.py
+++ b/test/image/test_load_image.py
@@ -63,7 +63,7 @@ def test_share_images_cache(tmp_path):
 
     with time_execution() as duration:
         build_pdf_with_big_images()
-    first_time_duration =  duration.seconds
+    first_time_duration = duration.seconds
 
     with time_execution() as duration:
         build_pdf_with_big_images()

--- a/test/image/test_load_image.py
+++ b/test/image/test_load_image.py
@@ -63,8 +63,8 @@ def test_share_images_cache(tmp_path):
 
     with time_execution() as duration:
         build_pdf_with_big_images()
-    assert duration.seconds > 0.25
+    first_time_duration =  duration.seconds
 
     with time_execution() as duration:
         build_pdf_with_big_images()
-    assert duration.seconds < 0.3
+    assert duration.seconds < first_time_duration / 2

